PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/appservice/component/input/AzureComboBox.java

```diff
@@ -123,18 +155,25 @@ protected void handleLoadingError(Throwable e) {
         }
     }
 
+    protected boolean isFilterable() {
+        return true;
+    }
+
     public void clear() {
         this.removeAllItems();
         this.setSelectedItem(null);
     }
 
     class AzureComboBoxEditor extends BasicComboBoxEditor {
         private Object item;
+        private ExtendableTextField textField;
 
         @Override
         public void setItem(Object item) {
             this.item = item;
-            this.editor.setText(getItemText(this.item));
+            if (!AzureComboBox.this.isPopupVisible()) {
```

Comment:
what would show in editor when popup is not visible?

Reply:
Show the item description, just keep quite when change selection in pop up status, in case trigger the filter